tasks.py

```diff
@@ -0,0 +1,198 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import json
+import os
+
+from invoke import Collection, task, run
+
+DOCKER_IMAGE_NAME = os.getenv('DOCKERHUB_REPO', 'lando-ui')
+project_root = os.path.dirname(__file__)
+
+# Name used by docker-compose to create a test-only docker environment.
+project_test_name = 'testlandoui'
+
+
+@task(name='remove-containers')
+def remove_containers(ctx):
+    """Remove all temporary containers created for testing."""
+    if not ctx.config.get('keep_containers'):
+        cmd = (
+            'docker-compose'
+            ' -f {project_root}/docker-compose.yml'
+            ' -p {test_project_name}'
+        ).format(
+            project_root=project_root, test_project_name=project_test_name
+        )
+
+        ctx.run(cmd + ' stop', pty=True, echo=True)
+        ctx.run(cmd + ' rm --force -v', pty=True, echo=True)
+
+
+@task(
+    name='python',
+    help={
+        'testargs': 'Arguments to pass to the test suite (default: \'\')',
+        'keep': 'Do not remove the test container after running',
+    },
+    post=[remove_containers]
+)
+def test_python(ctx, testargs='', keep=False):
+    """Run lando-ui python tests."""
+    ctx.config.keep_containers = keep  # Stashed for our cleanup tasks
+    run(
+        'docker-compose'
+        ' -f {project_root}/docker-compose.yml'
+        ' -p {test_project_name}'
+        ' run'
+        '{rm}'
+        ' lando-ui'
+        ' pytest {args}'
+        ''.format(
+            project_root=project_root,
+            test_project_name=project_test_name,
+            args=testargs,
+            rm=('' if keep else ' --rm')
+        ),
+        pty=True,
+        echo=True
+    )
+
+
+@task(
+    name='javascript',
+    help={
+        'testargs': 'Arguments to pass to the test suite (default: \'\')',
+        'keep': 'Do not remove the test container after running',
+        'pty': 'Execute tests with a pty. Default: True',
```

Comment:
The default argument part of this help-string should be the same as the help-string of `testargs` above.